lib/QoreFtpClient.cpp

```diff
@@ -553,162 +576,162 @@ struct qore_ftp_private {
    }
    */
 
-   // unlocked
-   // RFC 2428 Extended Passive Mode
-   DLLLOCAL int connectDataExtendedPassive(ExceptionSink* xsink) {
-      // try extended passive mode
-      int code;
-      QoreStringNode* mr = sendMsg(code, "EPSV", 0, xsink);
-      if (!mr) {
-         assert(*xsink);
-         return -1;
-      }
-
-      FtpResp resp(mr);
-      if ((code / 100) != 2)
-         return -1;
-
-      // ex: 229 Entering Extended Passive Mode (|||63519|)
-      // get port for data connection
-      //printd(5, "EPSV: %s\n", resp.getBuffer());
+    // unlocked
+    // RFC 2428 Extended Passive Mode
+    DLLLOCAL int connectDataExtendedPassive(ExceptionSink* xsink) {
+        // try extended passive mode
+        int code;
+        QoreStringNode* mr = sendMsg(code, "EPSV", 0, xsink);
+        if (!mr) {
+            assert(*xsink);
+            return -1;
+        }
 
-      const char* s = strstr(resp.getBuffer(), "|||");
-      if (!s) {
-         xsink->raiseException("FTP-RESPONSE-ERROR", "cannot find port in EPSV response: %s", resp.getBuffer());
-         return -1;
-      }
-      s += 3;
-      char* end = (char* )strchr(s, '|');
-      if (!end) {
-         xsink->raiseException("FTP-RESPONSE-ERROR", "cannot find port in EPSV response: %s", resp.getBuffer());
-         return -1;
-      }
-      *end = '\0';
+        FtpResp resp(mr);
+        if ((code / 100) != 2)
+            return -1;
 
-      int data_port = atoi(s);
-      if (data.connectINET(host, data_port, timeout_ms, xsink)) {
-         if (!*xsink)
-            xsink->raiseErrnoException("FTP-CONNECT-ERROR", errno, "could not connect to extended passive data port (%s:%d)", host, data_port);
-         return -1;
-      }
-      printd(FTPDEBUG, "EPSV connected to %s:%d\n", host, data_port);
+        // ex: 229 Entering Extended Passive Mode (|||63519|)
+        // get port for data connection
+        printd(FTPDEBUG, "EPSV: %s\n", resp.getBuffer());
```

Comment:
is it intended to have this uncommented?

Reply:
yes, `printd()` is changed to nothing in the preprocessor with non-debug builds